keyboards/mntre_v3/keymaps/default/readme.md

```diff
@@ -0,0 +1,19 @@
+# Default layout of MNT Reform USB Keyboard (Standalone)
+
+This is exactly the same layout as in the original firmware.
+Backlight level can be decreased and increased using Fn+F1 and Fn+F2
+respectively.
+
+  ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+  ┃  Esc │  F1  │  F2  │  F3  │  F4  │  F5  │  F6  │  F7  │  F8  │  F9  │  F10 │  F11 │  F12 │    Fn    ┃
+  ┠──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────────┨
+  ┃  `~  │  1!  │  2@  │  3#  │  4$  │  5%  │  6^  │  7&  │  8*  │  9(  │  0)  │  -_  │  =+  │ Backspace┃
+  ┠──────┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──────┨
+  ┃    Tab   │   Q  │   W  │   E  │   R  │   T  │   Y  │   U  │   I  │   O  │   P  │  [{  │  ]}  │  \|  ┃
+  ┠──────────┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──────┨
+  ┃    LCtrl    │   A  │   S  │   D  │   F  │   G  │   H  │   J  │   K  │   L  │  ;:  │  '"  │   Enter  ┃
+  ┠─────────┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───────┨
+  ┃  LShift │  Del │   Z  │   X  │   C  │   V  │   B  │   N  │   M  │  ,<  │  .>  │  /?  │  Up  │ RShift┃
+  ┠─────────┼──────┴──┬───┴──────┼──────┴──┬───┴──────┴──┬───┴──────┼──────┼──────┼──────┼──────┼───────┨
+  ┃   RGUI  │   LGUI  │   ALT    │   Space │     Space   │  Space   │ ALT │ RCtrl │ Left │ Down │  Right┃
+  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
```

Comment:
```suggestion
    ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
    ┃  Esc │  F1  │  F2  │  F3  │  F4  │  F5  │  F6  │  F7  │  F8  │  F9  │  F10 │  F11 │  F12 │    Fn    ┃
    ┠──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────┼──────────┨
    ┃  `~  │  1!  │  2@  │  3#  │  4$  │  5%  │  6^  │  7&  │  8*  │  9(  │  0)  │  -_  │  =+  │ Backspace┃
    ┠──────┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──┴───┬──────┨
    ┃    Tab   │   Q  │   W  │   E  │   R  │   T  │   Y  │   U  │   I  │   O  │   P  │  [{  │  ]}  │  \|  ┃
    ┠──────────┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──────┨
    ┃    LCtrl    │   A  │   S  │   D  │   F  │   G  │   H  │   J  │   K  │   L  │  ;:  │  '"  │   Enter  ┃
    ┠─────────┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───┴──┬───────┨
    ┃  LShift │  Del │   Z  │   X  │   C  │   V  │   B  │   N  │   M  │  ,<  │  .>  │  /?  │  Up  │ RShift┃
    ┠─────────┼──────┴──┬───┴──────┼──────┴──┬───┴──────┴──┬───┴──────┼──────┼──────┼──────┼──────┼───────┨
    ┃   RGUI  │   LGUI  │   ALT    │   Space │     Space   │  Space   │ ALT │ RCtrl │ Left │ Down │  Right┃
    ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
```

Reply:
thanks for the suggestion i have refactored it:

<img width="769" alt="image" src="https://github.com/qmk/qmk_firmware/assets/44367581/e9f99f1a-159e-49cf-b89a-9091b99792c2">